.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
can you please also delete https://github.com/voxpupuli/puppet-powerdns/blob/main/.github/stale.yml and https://github.com/voxpupuli/puppet-powerdns/blob/main/.github/workflows/main.yml ?

Reply:
Done. I've also removed `spec/acceptance/nodesets/*`